torch/csrc/jit/passes/onnx/helper.cpp

```diff
@@ -186,7 +186,16 @@ Node* transformToONNXConcatNode(
   for (auto* input : lc_node->inputs()) {
     auto new_input =
         need_new_input ? g->addInput()->copyMetadata(input) : input;
-
+    // ONNX Concat requires tensor inputs with rank > axis(=0). Certain inputs from
```

Comment:
`rank > axis(=0)`

this looks a bit weird. What does `axis(=0)` mean?

Reply:
Described it in a more understandable way